DassHydro-dev/smash
# ENC: add io method for save and read model, and enhance pytest

Add IO method save/read model:
- Remove private F90 types: l_descriptor, u_descriptor, rowcol_to_ind_sparse, and local_active_cell
- Enable read/save the entire model

Tests:
- Add test for save/read model: check the values of certain attrs of model
- Fix diff_baseline: add equal_nan and decrease atol to 1e-3 (which are the min atol of all tests) for writing diff_baseline
- Enable tests for flow percentiles signatures (decrease atol to 1e-4).
